libtiledbsoma/CMakeLists.txt

```diff
@@ -187,10 +187,6 @@ if(MSVC)
 else()
   add_compile_options(-Wall -Wextra)
 
-  if(TILEDBSOMA_ENABLE_WERROR)
```

Comment:
Why is this being removed on this PR?

Reply:
It has been a while. Maybe try locally without it?  I think it may break.

Keep in mind that some people have settings newer than Ubuntu 22.04 and hence newer (pickier) compilers.  I believe we have a remnant compiler warning.

Redoing CMake in a follow-up is a TODO.

Reply:
> Maybe try locally without it? 

@eddelbuettel I'd like you to try locally without it.

Reply:
@johnkerl It is possible that I can't as it doesn't (fully) build for me as I may have communicated a few times.  I have libfmt9 leading to errors with `bld`.  Which is why I suggested you may try too.

Reply:
@johnkerl It seems to work here _for the more limited builds_ I do.  I can turn it back on, if stuff breaks we can deal with it.